.tekton/wait-for-image-task.yaml

```
@@ -0,0 +1,52 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: wait-for-image
+  namespace: rh-acs-tenant
+spec:
+  description: Waits until the specified image is found and returns its digest.
+  params:
+  - name: IMAGE
+    description: Image reference.
+    type: string
+  results:
+  - name: GIT_REF
+    description: Git commit for the image's source code if provided as `vcs-ref` image label.
+  - name: GIT_REPO
+    description: Git repository URL for the image's source code if provided as `source-location` image label.
```

Since these became non-optional, please update their descriptions
```suggestion
  - name: GIT_REF
    description: Git commit for the image's source code from `vcs-ref` image label.
  - name: GIT_REPO
    description: Git repository URL for the image's source code from `source-location` image label.
```